troubleshooting/debugging.md

```diff
@@ -1498,5 +1504,114 @@ If you use Amazon ECS FireLens, then standard out and standard error will be col
 }
 ```
 
+## Case Studies
+
+### 🚨 Warning 🚨
+
+The goal of the Case Studies section is to share useful insights and test results. These results however do not represent a guarantee of any kind. Your own results may vary significantly.
+
+Resource usage, error rates, and other findings vary based on many factors, including but not limited to: specific details in Fluent Bit configuration, the instance/runtime that a test was run on, the size and content of the log messages processed/parsed/sent, and various random factors including API and network latency. 
+
+### AWS for Fluent Bit Stability Tests
+
+The AWS Distro [stability tests](https://github.com/aws/aws-for-fluent-bit#aws-distro-for-fluent-bit-release-testing) are used to ensure our releases maintain a high quality bar and to select our stable version. 
+
+They use the [FireLens datajet project](https://github.com/aws/firelens-datajet) to run tasks on Amazon ECS that simulate real world scenarios. 
+
+#### Rate of network errors
+
+As noted in the [Common Network Errors](#common-network-errors) and [Network Connection Issues](#network-connection-issues) sections, Fluent Bit's http client is very low level and can emit many error messages during normal/typical operation. 
+
+If you experience these errors, the key is to look for [messages that indicate log loss](#how-do-i-tell-if-fluent-bit-is-losing-logs). These errors matter when they cause impact. 
+
+Users often ask us what is a "typical" rate of these errors. Please see, the [Case Studies Warning](#🚨-warning-🚨), there is not necessarily a "typical" rate. We want to be clear that we've seen test runs where the error rate was higher than is shown in these screenshots.
```

Comment:
What does the provided connection error diagrams tell me? We need to have a summary here 
- With X tasks with Y MB/s, what is the expected error rate?

Reply:
The whole point here is that there is no typical rate as I say, but that you might be surprised at how many errors you see, so here's an example. I'll add that.